File: base/src/main/java/de/hipp/pnp/base/converter/persistent/HashMapConverter.java
package de.hipp.pnp.base.converter.persistent;

import jakarta.persistence.Converter;
import org.springframework.stereotype.Component;

import java.util.HashMap;
import java.util.Map;

@Converter(autoApply = true)
@Component
public class HashMapConverter extends BaseConverter<Map<String, Object>> {

    public HashMapConverter() {
        this.type = new HashMap<String, Object>();
    }
}

File: base/src/main/kotlin/de/hipp/pnp/base/converter/persistent/HashMapConverter.kt
package de.hipp.pnp.base.converter.persistent

import jakarta.persistence.Converter
import org.springframework.stereotype.Component

@Converter(autoApply = true)
@Component
class HashMapConverter : BaseConverter<Map<String, Any>>() {
    init {
        this.type = HashMap<String, Any>()
    }
}
